packages/react/src/experimental/OneDataCollection/index.tsx

```diff
@@ -152,28 +197,46 @@ export const OneDataCollection = <
   Filters extends FiltersDefinition,
   Sortings extends SortingsDefinition,
   ItemActions extends ItemActionsDefinition<Record>,
+  NavigationFilters extends NavigationFiltersDefinition,
 >({
   source,
   visualizations,
   onSelectItems,
   onBulkAction,
 }: {
-  source: DataSource<Record, Filters, Sortings, ItemActions>
-  visualizations: ReadonlyArray<Visualization<Record, Filters, Sortings>>
+  source: DataSource<Record, Filters, Sortings, ItemActions, NavigationFilters>
+  visualizations: ReadonlyArray<
+    Visualization<Record, Filters, Sortings, ItemActions, NavigationFilters>
+  >
   onSelectItems?: OnSelectItemsCallback<Record, Filters>
   onBulkAction?: OnBulkActionCallback<Record, Filters>
 }): JSX.Element => {
   const {
+    // Filters
     filters,
     currentFilters,
     setCurrentFilters,
+    presets,
+    // Navigation filter
+    currentNavigationFilters,
+    navigationFilters,
+    setCurrentNavigationFilters,
+    // Search
     search,
     currentSearch,
     setCurrentSearch,
+    //
     isLoading,
+    // Actions
     primaryActions,
     secondaryActions,
-    presets,
+    // Summary
+    totalItemSummary = (totalItems: number | undefined) => {
+      if (totalItems === undefined) {
+        return null
+      }
+      return `${totalItems} ${i18n.collections.itemsCount}`
```

Comment:
I'd consider extracting this default setter to a function. 

Or maybe compressing it down to `return totalItems !== undefined ? `${totalItems} ${i18n.collections.itemsCount}` : null`

Reply:
I can not extract it to a function as depends in i18n which is defined inside the hook, but i can compress it in a single line

Reply:
true, my bad